preview/deps/allclasses-noframe.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
It would be easiest to go to the branch and see that we now have a "preview" folder and it contains content for preview documentation. If you drill down into each folder, there will be an index.html that will show the API version number.